build.sbt

```diff
@@ -225,6 +226,7 @@ lazy val V = new {
     Seq(
       scala213,
       scala212,
+      "2.12.14",
```

Comment:
You should change scala212 to be `2.12.14`, we try to keep on the latest version in Metals, and add `2.12.13` here. You will most likely need to set ammonite212Version to `2.12.14` if it's not yet published.

Also, please update `bin/test-release.sh`

Reply:
scalafix isn't published for 2.12.14 yet and fails with:
```
[error] error: The ExplicitResultTypes rule only supports the exact Scala versions 
'List(2.12.13, 2.12.11, 2.12.12)' for this binary version.
```

Reply:
ach, ok, we can update it later.

Reply:
and the same for ammonite